ale/util.py

```diff
@@ -495,7 +500,7 @@ def duckpool(naifvar, start=0, length=10, default=None):
     return default
 
 
-def query_kernel_pool(matchstr="*"):
+def query_kernel_pool(matchstr="*", max_length=10):
```

Comment:
Why is this being added? I could see some instances where we need more than 10 values.

Reply:
10 was the previous default and works in *most* situations but I needed to add if for a particular driver. I just can't remember if it was MVIC TDI or Marci